src/main/scala/notifications/kafka/Serdes.scala

```diff
@@ -0,0 +1,90 @@
+package notifications.kafka
+
+import com.lightbend.kafka.scala.streams.{Deserializer, ScalaSerde, Serializer}
+import notifications.{EmailNotification, JsonSupport, SlackNotification}
+import providers.openweathermap.Responses
+import spray.json.JsonParser.ParsingException
+import spray.json._
+
+object Serdes {
+
+  object Email extends ScalaSerde[EmailNotification] with JsonSupport {
+
+    final object Deserializer extends Deserializer[EmailNotification] {
+      override def deserialize(data: Array[Byte]): Option[EmailNotification] = {
+        try {
+          val jsonData = new String(data).parseJson
+          Some(jsonData.convertTo[EmailNotification])
+        }
+        catch {
+          case e: ParsingException => None
```

Comment:
is this really something we want? completely ignoring exceptions?

Reply:
no, it should at least log this somewhere but do we have the time